### PR TITLE
Added forge init cmd to README

### PR DESCRIPTION
**Pull request by mothran**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,11 @@ It provides a starting point for building powerful new applications on Ethereum
 
 ## Getting Started
 
-Start building your application by forking this template.
+Start building your application by forking this template. Or if you already have Foundry installed running:
+
+```bash
+forge init -t https://github.com/risc0/bonsai-foundry-template.git ./my-project
```
Comment (hashcashier) [inline README.md:14]: We can shorten this to:
```
forge init -t risc0/bonsai-foundry-template ./my-project
```

Comment (mothran) [reply README.md]: TIL! Thats so cool! resolved in 041311a702a109a8d8dd9904e3b3a183e66684ea

**Review by tzerrell — APPROVED** [review]
